### DISCUSSION TITLE
Developer Mode

### DISCUSSION THREAD

**Discussion by andrewamos** [unoplatform/uno#16878] [Q&A]
Hi everyone I'm brand new to the UNO platform and WPF in general. 
I fell at the first fence : Developer Mode;  why does UNO platform need to be in Developer Mode ? - my IT department are blowing their top and dragging their feet in letting me had open rights to my PC, how do other companies resolve this ?- my IT department only seem to be able to support minimum rights i.e. for office based apps.
Thanks A.

**Comment by jeromelaban** [comment]
@andrewamos developer mode is needed if you use plan to deploy packaged MSIX apps, using the `net8.0-windows10.xxx` target framework. Otherwise, it's not needed for the other platforms. We'll update the docs to reflect this particular restriction.

**Reply by jeromelaban** [reply]
@andrewamos This is generally caused by the `YourApp (WinAppSDK Packaged)` profile. There is a VS issue that causes the Packaged profile to select itself if there are iOS and Android targets in the project. The best way to work around this is to remove the `YourApp (WinAppSDK Packaged)` from the `launchSettings.json`, then select `YourApp (WinAppSDK Unpackaged)` to run.

**Comment by andrewamos** [comment]
Thanks  Jérôme,
I'm trying to run the examples but cannot as they require to have developer mode enabled, can I hack the solutions/project file to stop this ?


Many regards Andy


Andrew Amos
Senior Embedded Software Engineer
Elant House | Unit A3 | Old Power Way | Lowfields Business Park | HX5 9DE | Elland

​If you are not the named addressee you should not disseminate, distribute or copy this e-mail. Please notify the sender immediately by e-mail if you have received this e-mail by mistake and delete this e-mail from your system. If you are not the intended recipient you are notified that disclosing, copying, distributing or taking any action in reliance on the contents of this information is strictly prohibited.
​
Fathom3 Limited Registered in England & Wales. Registration number 12272043. Registered office, Elant House, Unit A3, Old Power Way, Lowfields Business Park, HX5 9DE, Elland.

**Comment by andrewamos** [comment]
Thanks I'll give it a try

Regards Andy


Andrew Amos
Senior Embedded Software Engineer
Elant House | Unit A3 | Old Power Way | Lowfields Business Park | HX5 9DE | Elland

​If you are not the named addressee you should not disseminate, distribute or copy this e-mail. Please notify the sender immediately by e-mail if you have received this e-mail by mistake and delete this e-mail from your system. If you are not the intended recipient you are notified that disclosing, copying, distributing or taking any action in reliance on the contents of this information is strictly prohibited.
​
Fathom3 Limited Registered in England & Wales. Registration number 12272043. Registered office, Elant House, Unit A3, Old Power Way, Lowfields Business Park, HX5 9DE, Elland.
